model/usage.go

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+type PostsUsage struct {
+	Count int `json:"count"`
```

Comment:
Max int is about 2.14 Billion. That's a huge number, but I think we should probably change this to int64.

Reply:
Done.